# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../anjuta/anjuta-2.32.1.1-no-update-mime-database.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- anjuta-2.32.1.1/mime/Makefile.am.orig	2011-05-19 14:17:18.852002344 +0200
+++ anjuta-2.32.1.1/mime/Makefile.am	2011-05-19 15:46:09.072002345 +0200
@@ -7,11 +7,6 @@
 ## Update mime only when system wide installation takes place
 
 install-data-local: install-mimexmlDATA
-	if [ -f $(DESTDIR)$(datadir)/mime/packages/freedesktop.org.xml ] ; then \
-	  if which update-mime-database>/dev/null 2>&1; then \
-	    update-mime-database $(DESTDIR)$(datadir)/mime; \
-	  fi \
-	fi
 
 uninstall-local:
 	if [ -f $(DESTDIR)$(datadir)/mime/packages/freedesktop.org.xml ] ; then \
